core/chain_manager.go

```diff
@@ -48,6 +48,9 @@ var (
 )
 
 const (
+	headerCacheLimit    = 512
+	bodyCacheLimit      = 256
+	tdCacheLimit        = 1024
```

Comment:
But that would only benchmark the imports and not the network effect when people are sending blocks all over the place and requesting all kinds of information while they themselves are syncing.